### DISCUSSION TITLE
part-ii/2015-13/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#419] [Question comments]
Part II 2015 Applied Probability: Paper 3, Section II, K 

https://questions.tripos.org/part-ii/2015-13/

**Comment by ArthurConmy** [comment]
The first part is bookwork.

For the second part,
<details>
<summary>Spoiler</summary>

By the first part we can consider $n$ independent $\text{Poi}(1)$ random variables, $X_1, ... , X_n$. Then by the union bound

$$ \mathbb{ P}( X_1 , ... , X_n \ge x) \le n \mathbb{ P}(X_1 \ge x) \le n \text{exp}(x-x\log x) =: p_n$$

and now choosing $x = (1+\varepsilon )\frac{\log n}{\log \log n}$ works, because for all $C>0$, we can show that $p_n \le C $ eventually, as 

$$ \log p_n \le \log n - (1+\varepsilon ) \frac{\log n}{\log \log n}  \times ( \log \left( \frac{\log n}{ \log \log n }  \right)  - 1 ) \to 0$$

because the $-\varepsilon \log n$ term dominates.
</details>
